ginivision/src/main/res/layout/gv_fragment_camera.xml

```diff
@@ -41,6 +41,19 @@
         android:background="@null"
         android:src="@drawable/gv_camera_trigger" />
 
+    <ImageButton
+        android:id="@+id/gv_button_import_document"
+        android:layout_width="48dp"
+        android:layout_height="wrap_content"
+        android:layout_alignBottom="@+id/gv_button_camera_trigger"
+        android:layout_alignTop="@id/gv_button_camera_trigger"
+        android:layout_marginRight="36dp"
+        android:layout_toLeftOf="@id/gv_button_camera_trigger"
+        android:background="@null"
+        android:scaleType="centerInside"
+        android:src="@android:drawable/stat_sys_upload"
```

Comment:
don't know if this should be chosen since it might be animated on some devices(:
![serranoltexxkot49hstephan09132017112036](https://user-images.githubusercontent.com/434214/30369738-c8226770-9875-11e7-829f-f36f842ee0e6.gif)

Reply:
Congratulations, you found an easter egg 🎉 !

This is only a placeholder: `The UI is not finalized and no resources were used in the layouts. We will finalize it later.`